src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskClaimByIdControllerTest.java

```diff
@@ -102,6 +105,51 @@ public void user_should_claim_task_when_role_assignment_verification_passed() {
         common.cleanUpTask(taskId);
     }
 
+    @Test
+    public void user_should_claim_task_and_claim_action_recorded_in_replicate_db() throws InterruptedException {
+
+        TestVariables taskVariables = common.setupWATaskAndRetrieveIds("processApplication",
+                                                                       "Process Application");
+        initiateTask(taskVariables);
+
+        common.setupWAOrganisationalRoleAssignment(caseworkerCredentials.getHeaders(), "tribunal-caseworker");
+
+        TimeUnit.SECONDS.sleep(10);
```

Comment:
Doing arbitrary waits like this are not good practice and leads to inconsistent tests when you run in different environments.  It is much better to poll for a condition.

Reply:
Removed wait as not required anymore.